bokehjs/src/coffee/models/plots/plot_canvas.coffee

```diff
@@ -210,6 +210,32 @@ export class PlotCanvasView extends DOMView
     follow_enabled = false
     has_bounds = false
 
+    if @model.plot.aspect_ratio != false
+      aspect = if @model.plot.aspect_ratio is true then 1 else @model.plot.aspect_ratio
+      r = aspect*(@model.plot.width/@model.plot.height)
+
+      for k, v of bounds
```

Comment:
I'm still a beginner in Coffescript, but is comprehension the correct way of looping through the object? Would closing the scope be preferred? i.e. 
```
for k, v of bounds 
  do(k, v) -> 
```